spyder_terminal/tests/test_terminal.py

```diff
@@ -131,3 +134,25 @@ def test_output_redirection(qtbot):
     stderr = osp.join(getcwd(), 'spyder_terminal_err.log')
     assert osp.exists(stdout) and osp.exists(stderr)
     terminal.closing_plugin()
+
+
+def test_close_terminal_manually(qtbot):
+    # Setup widget
+    terminal = setup_terminal(qtbot)
+    qtbot.wait(TERM_UP)
```

Comment:
@goanpeca If I use the ready signal, some tests start to fail randomly. I prefer to leave them as they are